app/interfaces/api/entities/scheme_ten_claim.rb

```diff
@@ -0,0 +1,15 @@
+module API
+  module Entities
+    class SchemeTenClaim < BaseEntity
```

Comment:
Following https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2339/files#r183688638, this class should probably be called `AgfsSchemeTenClaim ` or even put that as a namespace (`AgfsSchemeTen::Claim`), up to you ;)